circus-train-integration-tests/src/test/data/com/hotels/bdp/circustrain/integration/CircusTrainHdfsS3IntegrationTest/unpartitioned-single-table-avro-schema.yml

```diff
@@ -0,0 +1,10 @@
+table-replications:
+  - source-table:
+      database-name: ${circus-train-runner.database-name}
+      table-name: ct_table_u
+    replica-table:
+      table-name: ct_table_u_copy
+      table-location: s3a://replica/${circus-train-runner.database-name}/ct_table_u_copy
```

Comment:
Is there a reason we are using `s3a` here rather than just `s3`?

Reply:
I have no clue but it doesn't work if I use s3, so I stuck with the going examples. I suspect something in s3Proxy test framework. Not ideal haven't explored it further.

Reply:
OK, weird, let's leave it like this for now then.